tests/test_pytest/test_plugin_schema/list.yaml

```diff
@@ -0,0 +1,26 @@
+interface: products
+family: list
+name: abi
+docstring: >
+  List of products for the ABI sensor
+
+  A list of products for the Advanced Baseline Imager on the GOES satellites.
+spec:
+  - interface: products
```

Comment:
This works with no spec defined for the "infrared" ABI product?

Reply:
This is a bad file that wasn't actually being tested. I've deleted it.